docs/api/commands/window.mdx

```diff
@@ -31,10 +31,26 @@ Pass in an options object to change the default behavior of `cy.window()`.
 | `log`     | `true`                                                               | Displays the command in the [Command log](/guides/core-concepts/cypress-app#Command-Log) |
 | `timeout` | [`defaultCommandTimeout`](/guides/references/configuration#Timeouts) | Time to wait for `cy.window()` to resolve before [timing out](#Timeouts)                 |
 
+### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
+
+- `cy.window()` ignores the subject it is given.
+
+### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
+
+- `cy.window()` has no implicit assertions.
+
+### Timeouts [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Timeouts)
+
+- `cy.window()` cannot time out. The `timeout` option is only useful to set the
+  timeout of following assertions.
+
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
+- `cy.window()` is a query command, and is
```

Comment:
I'd expect this to be the second bullet after we say what it yields

Reply:
So the reason I put the "is a query command" first is that some commands have long `yields` sections. Eg:

> ![image](https://user-images.githubusercontent.com/3003404/224115364-30f06db8-c325-4e42-84f0-613d60e659ca.png)

------

I was aiming to always have it in the same place (eg, first or second), and in the cases where we need to spend time explaining what's yielded in more detail, having it second ended up making it feel buried under the other text.

> ![image](https://user-images.githubusercontent.com/3003404/224115908-97f9824a-0f2b-4ce6-97ba-f438b6b2674a.png)

-------

I'd be fine breaking the consistency here, having it sometimes below what's actually yielded (you're right it is better in the `short case`) and sometimes above (I do think it needs to come first in the 'long case'). Thoughts?